### PR TITLE
[MU3] Fixed resetting of styles

### PR DESCRIPTION
**Pull request by RomanPudashkin**
implemented "Reset All Styles to Default", "Reset Text Style Overrides" and "Reset All Pages Size to Default" buttons

resolves: https://trello.com/c/5vVGay6d/67-cannot-globally-reset-text-styles-to-edwin-for-imported-xml-documents

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Is that related to #7127 ?

File: mscore/pagesettings.ui [hunk 1]
```diff
@@ -247,6 +247,13 @@
      <property name="bottomMargin">
       <number>0</number>
      </property>
+     <item>
+      <widget class="QPushButton" name="resetPageStyleButton">
+       <property name="text">
+        <string>Reset All Page Sizes to Default</string>
```
Comment (shoogle) [inline mscore/pagesettings.ui:253]: String should be: "Reset All Page *Settings* to Default"

Comment (RomanPudashkin) [reply mscore/pagesettings.ui:253]: I think we need the help of @Tantacrul here. What text should be in the button for resetting the page styles (this button is located in Format -> Page Settings)?

Comment (shoogle) [reply mscore/pagesettings.ui:253]: The button string should match the name of the dialog, like:

- Preferences dialog: "Reset All Preferences to Default"
- Style dialog: "Reset All Styles to Default"

Unless there are some things in the Page Settings dialog that are not affected by the reset?

File: libmscore/cmd.cpp [hunk 2]
```diff
@@ -4173,7 +4173,7 @@ void Score::cmd(const QAction* a, EditData& ed)
             { "pad-note-512-TAB",           [](Score* cs, EditData& ed){ cs->padToggle(Pad::NOTE512, ed);                             }},
             { "pad-note-1024",              [](Score* cs, EditData& ed){ cs->padToggle(Pad::NOTE1024, ed);                            }},
             { "pad-note-1024-TAB",          [](Score* cs, EditData& ed){ cs->padToggle(Pad::NOTE1024, ed);                            }},
-            { "reset-style",                [](Score* cs, EditData&){ cs->cmdResetStyle();                                            }},
+            { "reset-style",                [](Score* cs, EditData&){ cs->cmdResetAllStyle();                                            }},
```
Comment (Jojo-Schmitz) [inline libmscore/cmd.cpp:4176]: minor nitpick: whitespace damage

File: mscore/pagesettings.ui (hunk 1 above)
Comment (RomanPudashkin) [reply mscore/pagesettings.ui]: Changes text of this button, thanks

File: libmscore/cmd.cpp (hunk 2 above)
Comment (RomanPudashkin) [reply libmscore/cmd.cpp]: fixed, thanks

**Comment by RomanPudashkin** [comment]
> Is that related to #7127 ?

I don't know :) The main purpose of this PR is to enable applying Edwin for imported scores (from XML)

**Pull request merged**